dist/debian/rules

```diff
@@ -74,11 +76,14 @@ override_dh_auto_configure:
 	@export GOPATH=$(GOROOT)/bin PATH=$(GOROOT)/bin:$$PATH GOCACHE=$(GOCACHE); \
 	  set -ex; \
 	  if ! ./mlocal/scripts/check-min-go-version go $(MINGO_VERSION); then \
-	    if [ -d $(GOROOT) ]; then rm -rf $(GOROOT); fi; \
+	    wget -nc -O debian/$(TARBALL) https://dl.google.com/go/$(TARBALL); \
```

Comment:
Doesn't this go against the requirement of not using the network during package build?  I had assumed the golang tarball would be supplied externally.

Reply:
The `.spec.in` is the manually maintained source of truth for where to get those packages.  Everything else derives from there.

A script that is used in common in rpm and debian package building to download if necessary, unpack, and compile go is probably a good idea.

I tested `wget -nc` and you're right that if the file already exists it does not even attempt to go out to the internet to fetch it.  You would need to make sure that wget is a build-time dependency for both rpm & debian.